### PR TITLE
[vscode] VSCode Syntax Highlighting for Virgil

### PR DESCRIPTION
**Pull request by btwj**
Visual Studio Code syntax highlighting for Virgil
- Uses tree-sitter to parse Virgil files, and walks the AST to output semantic tokens

### CODE REVIEW

**Review by titzer — APPROVED** [review]
lgtm, mostly rubber-stamping.

File: bin/v3c [hunk 1]
```diff
@@ -1,9 +0,0 @@
-#!/bin/bash
```
Comment (k-sareen) [inline bin/v3c:1]: You've accidentally checked in your version of `bin/v3c`.

Comment (titzer) [reply bin/v3c:1]: The incantation to fix this is `git update-index --assume-unchanged bin/v3c`.

Comment (k-sareen) [reply bin/v3c:1]: Why not just add it to `.gitignore`? Unlikely that the script will have any changes, it's pretty simple.

Comment (titzer) [reply bin/v3c:1]: Hmm, I just tried that and it doesn't want to ignore it. I guess `.gitignore` doesn't really apply to checked-in files.

Comment (k-sareen) [reply bin/v3c:1]: Hm yeah. I just tried it too and it didn't work. Oh well.

**Review by titzer — APPROVED** [review]
lgtm

**Comment by titzer** [comment]
@btwj If you want to mark this as no longer a draft, I'll merge it.

**Comment by titzer** [comment]
> @btwj If you want to mark this as no longer a draft, I'll merge it.

Nm, I have the power. Going to merge. Feel free to send follow up PRs.

**Pull request merged**